plugin/fireplace.vim

```diff
@@ -1666,6 +1666,42 @@ augroup fireplace_go_to_file
   autocmd FileType clojure call s:set_up_go_to_file()
 augroup END
 
+" Section: Spec
+" TODO: `spec-list`
+
+function! fireplace#spec(op, symbol) abort
+  if fireplace#op_available(a:op)
+    let unaliased_symbol = fireplace#eval(a:symbol)
```

Comment:
Eval is a really crude hammer to swing here. Try something like this:
```vim
  if a:symbol =~# '^::.\+/'
    let symbol = ':' . fireplace#resolve_alias(matchstr(a:symbol, '^::\zs[^/]\+')) . matchstr(a:symbol, '/.*')
  elseif a:symbol =~# '^::'
    let symbol = ':' . fireplace#ns() . '/' . strpart(a:symbol, 2)
  else
    let symbol = a:symbol
  endif
```

It won't handle `(alias 'self *ns*)`, but I'll worry about that when I see it in production code.

Reply:
Just for the records: `(alias 'self *ns*)` is just a hack to have a single file but be able to test `::nsalias/kw`, which is became way more likely since spec.  More realistic example

```clojure
(ns com.example.game.board (:require [com.example.game.player :as player] ...))
(s/def ::board (s/keys :req [::player/players]))
```

Reply:
Exactly what I figured. I think we're good.